### PR TITLE
Nexla ai nova

### PR DESCRIPTION
**Pull request by Noel1997**
## Checks

- [ ] I've included any doc changes needed for https://autogenhub.github.io/autogen/. See https://autogenhub.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: website/blog/2024-10-23-NOVA/index.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Unlocking the Power of Agentic Workflows at Nexla with Autogen
+authors:
+  - Noel1997
+  - ameydesai
+tags: [data automation, agents, Autogen, Nexla]
+---
+
```
Comment (sonichi) [inline website/blog/2024-10-23-NOVA/index.mdx:8]: Inserting an image here would increase the interest of readers. Optional.

**Review by sonichi — APPROVED** [review]

File: website/blog/2024-10-23-NOVA/index.mdx (hunk 1 above)
Comment (Noel1997) [reply website/blog/2024-10-23-NOVA/index.mdx:8]: Added

File: website/blog/2024-10-23-NOVA/index.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## Natural Language to Transforms
+
+NOVA’s Natural Language to Transforms feature allows users to take a Nexset—a data product within Nexla—and describe, in plain language, the transformation they need. NOVA then automatically generates the required transforms, whether in Python or SQL, depending on the task.
+
+For example, a user could simply instruct, "Compute average speed and average duration for every origin-destination pair, hourly and by day of the week." NOVA breaks down this request into a series of steps, applies the necessary transformations, and delivers the desired output. This allows users to focus on analyzing and utilizing the transformed data without getting bogged down in the complexities of coding.
+
+## Natural Language to ELT
+
+Next up is Natural Language to ELT, which allows users to build and execute ELT pipelines simply by providing natural language instructions. Users can input one or more Nexsets, a final data model, and an optional set of instructions, and NOVA does the rest.
```
Comment (sonichi) [inline website/blog/2024-10-23-NOVA/index.mdx:35]: Minor: Should ELT be ETL? Is it good to provide the full name in the first occurrence?

**Review by sonichi — APPROVED** [review]

**Review by qingyun-wu — APPROVED** [review]
Thanks Noel! Looks good to me!

**Pull request merged**